vmdb/app/models/ems_kubernetes.rb

```diff
@@ -1,8 +1,4 @@
 class EmsKubernetes < EmsContainer
-  has_many :container_nodes,                      :foreign_key => :ems_id, :dependent => :destroy
```

Comment:
I'd be surprised if you can't assign a specific entry-point (service) to an amazon container group (task). Even in that remote possibility it just means that the provider has an empty list of services.

Anyway moving it to a mixin is a good idea